sql/sql_table.h

```diff
@@ -251,6 +253,11 @@ bool quick_rm_table(THD *thd, handlerton *base, const LEX_CSTRING *db,
                     const LEX_CSTRING *table_name, uint flags,
                     const char *table_path=0);
 void close_cached_table(THD *thd, TABLE *table);
+Create_field *mysql_add_invisible_field(THD *thd, List<Create_field> * field_list,
+                              const char *field_name,
+                              const Type_handler *type_handler,
+                              field_visibility_t invisible, Item* default_value,
+                              bool force_gen_name);
```

Comment:
Wait, don't you use all those functions only in sql_table? I think if it's the case, then better to leave them static